docker.env

```diff
@@ -2,6 +2,5 @@ STARKNET_NETWORK=alpha-goerli
 STARKNET_WALLET=starkware.starknet.wallets.open_zeppelin.OpenZeppelinAccount
 STARKNET_ACCOUNT_DIR=/usr/src/warp/.starknet_accounts_devnet
 
-# Doesn't work (Cairo 0.10.0, 0.10.1)
-# STARKNET_GATEWAY_URL=http://devnet:5050
-# STARKNET_FEEDER_GATEWAY_URL=http://devnet:5050
+STARKNET_GATEWAY_URL=http://devnet:5050
+STARKNET_FEEDER_GATEWAY_URL=http://devnet:5050
```

Comment:
Are you sure it is fixed? I've created [the issue](https://github.com/starkware-libs/cairo-lang/issues/118) in cairo repository and it is not resolved

Reply:
Warp now has the feeder gateway url and the gateway url and it picks them up from the environment and passes them explicitly to the starknet cli